app/views/application/interview/index.njk

```diff
@@ -18,17 +18,26 @@
 
 {% block primary %}
 
-  <p class="govuk-body">Your interview will usually take place over the course of a day and you will need to attend in person. Please give details of:</p>
+  <p class="govuk-body">Interviews usually take place over the course of a day. You'll need to attend in person.</p>
+
+  <p class="govuk-body">Your training provider(s) might not have much flexibility when setting a date and time for interview.</p>
+
+  <p class="govuk-body">If there's a reason why you need some flexibility you can tell us about it here.</p>
+
+  <p class="govuk-body">For example:</p>
   <ul class="govuk-list govuk-list--bullet">
-    <li>any disability, health or personal issue you feel is relevant</li>
-    <li>special arrangements or assistance you may need</li>
-    <li>dates you are unavailable for interview</li>
+    <li>you have commitments such as employment or caring responsibilities</li>
+    <li>you'll be travelling a long way to get to the interview</li>
+    <li>you'll need some adjustments because you're disabled</li>
+    &nbsp
```

Comment:
This should be removed and the `<ul>` put back where it was. Let's chat through how to add space. :)